projects/igniteui-angular/src/lib/chips/chips-area.spec.ts

```diff
@@ -93,6 +174,576 @@ describe('IgxChipsArea', () => {
+        const secondChipElem = secondChip.chipArea.nativeElement;
+
+        const startingTop = secondChipElem.getBoundingClientRect().top;
+        const startingLeft = secondChipElem.getBoundingClientRect().left;
+        const startingBottom = secondChipElem.getBoundingClientRect().bottom;
+        const startingRight = secondChipElem.getBoundingClientRect().right;
+
+        const startingX = (startingLeft + startingRight) / 2;
+        const startingY = (startingTop + startingBottom) / 2;
+
+        simulatePointerEvent('pointerdown', secondChipElem, startingX, startingY);
+        fix.detectChanges();
+
+        fix.whenStable().then(() => {
+            fix.detectChanges();
+            simulatePointerEvent('pointermove', secondChipElem, startingX + 10, startingY + 10);
+
+            return fix.whenStable();
+        }).then(() => {
+            fix.detectChanges();
+            simulatePointerEvent('pointermove', secondChip.dragDir['_dragGhost'], startingX + xDragDifference, startingY + yDragDifference);
+            fix.detectChanges();
+
+            setTimeout(() => {
+                expect(secondChipElem.style.visibility).toEqual('hidden');
+            });
+        });
+
+    }));
+
+    it('all suffix conectors should be visible after chip is dropped', async(() => {
+        const xDragDifference = 200;
+        const yDragDifference = 100;
+        const fix = TestBed.createComponent(TestChipComponent);
+        fix.detectChanges();
+
+        const chipArea = fix.debugElement.queryAll(By.directive(IgxChipsAreaComponent));
+        const chipComponents = chipArea[0].queryAll(By.directive(IgxChipComponent));
+        const secondChip = chipComponents[1].componentInstance;
+        secondChip.dragDir.animateOnRelease = false;
+        const secondChipElem = secondChip.chipArea.nativeElement;
+
+        const startingTop = secondChipElem.getBoundingClientRect().top;
+        const startingLeft = secondChipElem.getBoundingClientRect().left;
+        const startingBottom = secondChipElem.getBoundingClientRect().bottom;
+        const startingRight = secondChipElem.getBoundingClientRect().right;
+
+        const startingX = (startingLeft + startingRight) / 2;
+        const startingY = (startingTop + startingBottom) / 2;
+
+        simulatePointerEvent('pointerdown', secondChipElem, startingX, startingY);
+        fix.detectChanges();
+
+        fix.whenStable().then(() => {
+            fix.detectChanges();
+            simulatePointerEvent('pointermove', secondChipElem, startingX + 10, startingY + 10);
+
+            return fix.whenStable();
+        }).then(() => {
+            fix.detectChanges();
+            simulatePointerEvent('pointermove', secondChip.dragDir['_dragGhost'], startingX + xDragDifference, startingY + yDragDifference);
+            fix.detectChanges();
+
+            return fix.whenRenderingDone();
+        }).then(() => {
+            expect(secondChipElem.style.visibility).toEqual('hidden');
```

Comment:
This tests the chip base elements themselves, not the connectors. Same as the next check.